external/types/pdf.d.ts

```diff
@@ -78,6 +78,8 @@ declare module 'pdfjs-dist' {
 
   //
   // web/pdf_viewer.component.js
+  // Warning: These functions are only accessible with web/pdf_viewer.js.
+  //          It is NOT automatically imported with 'pdfjs-dist'.
```

Comment:
Unfortunately this is simply *not* correct, since the variables added below are not exposed *anywhere*! And certainly not on the, [soon to be removed](https://github.com/mozilla/pdf.js/projects/5), global `PDFJS` object.

The variables added here correspond to https://github.com/mozilla/pdf.js/blob/master/web/default_preferences.json, which are *only* ever used in the default viewer (i.e. the one found at https://mozilla.github.io/pdf.js/web/viewer.html) and nowhere else. These are *not* used/exposed either through the PDF.js API, nor through https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.component.js. Hence the default viewer, i.e. file `web/app.js`, shouldn't be part of any "public" API.